app/src/main/java/org/stepic/droid/model/Device.kt

```diff
@@ -7,7 +7,8 @@ data class Device(
         var registration_id: String? = null,
         var user: Long = 0,
         var description: String? = null,
-        var client_type: ClientType
+        var client_type: ClientType,
+        val is_badges_enabled: Boolean?
```

Comment:
Let's use camelCase for this class with annotation `@SerializedName`. Later, we can set              `FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES` for gson and remove annotations.